qiskit/tools/jupyter/backend_monitor.py

```diff
@@ -30,7 +30,8 @@
     # pylint: disable=import-error
     from qiskit.providers.ibmq import IBMQBackend
 except ImportError:
-    pass
+    raise ImportError('Could not import the IMBQ provider from the qiskit-ibmq-provider'
```

Comment:
In this case, the `ImportError` is not necessary. (We check HAS_IBMQ in jupyter/__init__.py before we register _backend_monitor.)